apps/sms/views/conversation/js/compose.js

```diff
@@ -406,6 +406,14 @@ var Compose = (function() {
         subject.show();
       }
 
+      if (!draft.content) {
```

Comment:
__unrelated note:__ it's funny how at the beginning of the function we clear composer and only then we check if draft/draft content is valid :)